auth-server/test/resolvers/mutation/changeName.spec.ts

```diff
@@ -0,0 +1,58 @@
+import 'mocha'
+import { expect } from 'chai'
+import signup from '../../../src/resolvers/mutation/signup'
+import changeName from '../../../src/resolvers/mutation/changeName'
+import User from '../../../src/model/User'
+import { Context } from '../../../src/types'
+import { ForbiddenError } from 'apollo-server'
+
+describe('changeName mutation', () => {
+	let signupResult
+	const fakectx: Context = {
+		req: {
+			headers: {}
+		},
+		res: {
+			cookie: () => {}
+		}
+	} as any
+	const email = 'test@email.com'
+	const password = 'testpass'
+	const username = 'testuser'
+	const name = 'test name'
+
+	before(async () => {
+		signupResult = await signup(null, { email, password, username, name }, fakectx)
+		fakectx.req.headers.authorization = `Bearer ${signupResult.token}` // eslint-disable-line
+	})
+
+	after(async () => {
+		await User
+			.query()
+			.where({ id: signupResult.user.id })
+			.del()
+	})
+
+	it('should be able to change name', async () => {
+		const newName = 'new name'
+		await changeName(null, { newName }, fakectx)
+
+
+		const dbUser = await User
+			.query()
+			.where({ id: signupResult.user.id })
+			.first()
+
+		expect(dbUser.name).to.be.equal(newName)
+	})
+
+	it('should not be able to change name with fake token', async () => {
+		const newName = 'new name'
+		fakectx.req.headers.authorization = 'Bearer fake'
+		try {
+			await changeName(null, { newName }, fakectx)
+		} catch (error) {
+			expect(error).to.exist
```

Comment:
to use `ForbiddenError` and be consistent with other tests.
```suggestion
			expect(error).to.exist
			expect(error).to.be.an.instanceof(ForbiddenError)
```

Reply:
If token is fake then JWT library will throw error which is not forbidden error.

Reply:
Interesting to point out, this would probably be an unhandled promise error then, we need to catch those errors in `verify` and handle them properly. I'll create an issue.